src/Mod/PartDesign/App/Body.cpp

```diff
@@ -190,6 +189,9 @@ bool Body::isSolidFeature(const App::DocumentObject *obj)
         return false;
     }
 
+    // if (obj == BaseFeature.getValue())

```

Comment:
Should this be left commented out?

Reply:
Seems it's just an optimization that I brought over during code compare; makes sense to put it in.

Reply:
I take that back.  It doesn't work unless the method is converted from static, which triggers an avalanche of other issues.  That's why I originally removed the optimization, and now I've deleted it completely.